### DISCUSSION TITLE
How to set the parameter MaxMessageBytes?

### DISCUSSION THREAD

**Discussion by RusDenS** [jaegertracing/jaeger#3017] [Q&A]
Hi,

We encountered an error in our production environment:
_{"level":"error","ts":1616681027.718327,"caller":"kafka/writer.go:58","msg":"kafka server: Message was too large, server rejected it to avoid allocation error.","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:58"}_

This problem has already been described in [**issues**](https://github.com/jaegertracing/jaeger/issues/1335).

To solve this problem need to increase the parameter MaxMessageBytes in the component Sarama.
Is it possible to do this using CLI Flags Jaeger-Collector?

P.S. We use Jaeger-Collector:1.18.1 and Sarama v1.22.2

**Comment by jpkrohling** [comment]
That doesn't seem to be supported, but I think this would be a good improvement to have. Would you like to contribute a PR for that?
